src/components/grid/ProjectGrid.vue

```diff
@@ -1,47 +1,68 @@
 <template>
   <div class="project-grid">
-    <div>
-      <div class="row items-center">
-        <h4>
-          {{ $t('page.home.project.title') }}
-        </h4>
+    <div class="row items-center">
+      <h4 class="q-ma-md q-mr-xl">
+        {{ $t('page.home.project.title') }}
+      </h4>
+      <q-btn
+        outline
+        no-caps
+        class="q-ml-sm"
+        color="primary"
+        icon="fa-solid fa-cloud-arrow-down"
+        :label="$t('actions.home.importProject')"
+        data-cy="import-project-button"
+        @click="DialogEvent.next({ type: 'open', key: 'ImportProject' })"
+      />
+    </div>
+    <div class="row">
+      <h5 class="text-secondary q-ma-md">
+        {{ $t('page.home.project.recent') }}
+      </h5>
+    </div>
+    <div class="row no-wrap">
+      <div class="row">
         <q-btn
           outline
           no-caps
-          class="q-ml-xl"
           color="primary"
-          icon="fa-solid fa-plus"
-          :label="$t('actions.home.newProject')"
+          class="square-button"
           data-cy="create-project-button"
           @click="DialogEvent.next({ type: 'open', key: 'CreateProject' })"
-        />
-        <q-btn
-          outline
-          no-caps
-          class="q-ml-sm"
-          color="primary"
-          icon="fa-solid fa-cloud-arrow-down"
-          :label="$t('actions.home.importProject')"
-          data-cy="import-project-button"
-          @click="DialogEvent.next({ type: 'open', key: 'ImportProject' })"
-        />
+        >
+          <template #default>
+            <div class="column items-center">
+              <q-icon
+                name="fa-solid fa-plus"
+                color="primary"
+                size="4rem"
+              />
+              <label>
+                {{ $t('page.home.project.create') }}
+              </label>
+            </div>
+          </template>
+        </q-btn>
       </div>
-      <div class="row">
-        <project-card
+      <div class="row items-center">
+        <div
           v-for="project in projects"
           :key="project.id"
-          :project="project"
-          class="q-ma-sm"
-        />
-      </div>
-      <div
-        v-if="projects.length === 0"
-        class="row text-center text-h6 text-grey empty-grid"
-        data-cy="project-grid-empty"
-      >
-        {{ $t('page.home.project.empty') }}
+          class="row q-mb-lg justify-center items-center col-2"
+        >
+          <project-card
+            :project="project"
+          />
+        </div>
       </div>
     </div>
+    <div
+      v-if="Object.keys(projects).length === 0"
```

Comment:
Only `projects.length === 0` since it's now an Array

Reply:
done